lib/stages/visWriter.hpp

```diff
@@ -89,8 +89,10 @@
  * @metric kotekan_viswriter_write_time_seconds
  *         The write time of the HDF5 writer. An exponential moving average over ~10
  *         samples.
- * @metric kotekan_viswriter_dropped_frame_total
- *         The number of frames dropped while attempting to write.
+ * @metric kotekan_viswriter_late_frame_total
+ *         The number of frames dropped while attempting to write as they are too late.
+ * @metric kotekan_viswriter_bad_dataset_frame_total
```

Comment:
I know you expect this not to happen too often, but let's keep an eye for how it behaves in the running system, just so it doesn't blow up out of control. At worst, we can block it from being indexed by Prometheus, but still have it available if you `curl` the metrics from an individual node.